base/Dockerfile

```
@@ -20,6 +20,7 @@ RUN apt-get update && \
     python3-venv \
     git-lfs \
     unzip \
+    vim \
```

```suggestion
    neovim \
```

You're the better expert when it comes to `vim`, but as far as I know, `neovim` has most of the features that `vim` has. Reason for the proposal: The installation of `vim` increases the image size by `41.8 MB`, `neovim` only by `30.3 MB`.